Makefile

```diff
@@ -276,6 +277,7 @@ run-abci:
 	-e NETWORK_ID=$(NETWORK_ID) \
 	-e CHAIN_ID=$(CHAIN_ID) \
 	-e PREFIX=$(prefix) \
+	-e TX_QUERY_HOSTNAME=$(TX_QUERY_HOSTNAME) \
```

Comment:
need to change container name in run-sgx-query-next to match
https://github.com/crypto-com/chain/blob/eb69f7e278331a40672b2de5fc992852cd4649a2/Makefile#L255

```
--name $(prefix)$(TX_QUERY_HOSTNAME) \
```

Reply:
Sorry, can you elaborate?

Reply:
if user provide ENV `TX_QUERY_HOSTNAME=somethingelse` to run make command,
chain-abci in `ci-scripts/run_chain_abci.sh` cannot connect to tx-query container as the container name is still sgx-query-next

Reply:
`TX_QUERY_HOSTNAME` serves different purpose than I expected
so it makes sense now